src/gui/sysManager.cpp

```diff
@@ -82,10 +82,25 @@ void FurnaceGUI::drawSysManager() {
           ImGui::EndDragDropTarget();
         }
         ImGui::TableNextColumn();
+        bool isNotCollapsed=true; // IM SORRY
```

Comment:
Rationale for apology?

Reply:
i don't remember
i will remove that comment